From aa8d9b26ae513956946cae612821fc4e04439e45 Mon Sep 17 00:00:00 2001
From: Calvin Johnson <calvin.johnson@nxp.com>
Date: Thu, 9 May 2019 18:11:07 +0530
Subject: [PATCH 36/40] armv8: ls1043a: change SoC device ID to detect 23 x 23
 package

Modify SVR_LS1043A to 0x879202(23x23 package) which is different from
default 0x879200(21x21).

This is a temporary hack and has to be supported in a standard way
across all SoC types. This SVR definition is used at different places.
Hence keeping the definition same.

Signed-off-by: Calvin Johnson <calvin.johnson@nxp.com>
---
 arch/arm/include/asm/arch-fsl-layerscape/soc.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/include/asm/arch-fsl-layerscape/soc.h b/arch/arm/include/asm/arch-fsl-layerscape/soc.h
index 66aa19b025..bc249f85a9 100644
--- a/arch/arm/include/asm/arch-fsl-layerscape/soc.h
+++ b/arch/arm/include/asm/arch-fsl-layerscape/soc.h
@@ -57,7 +57,7 @@ struct cpu_type {
 #endif
 #define SVR_WO_E		0xFFFFFE
 #define SVR_LS1012A		0x870400
-#define SVR_LS1043A		0x879200
+#define SVR_LS1043A		0x879202
 #define SVR_LS1023A		0x879208
 #define SVR_LS1046A		0x870700
 #define SVR_LS1026A		0x870708
-- 
2.17.1

